agent/log-drop/log-drop.go

```diff
@@ -0,0 +1,70 @@
+package log_drop
```

Comment:
Could we follow [Effective Go's guidance](https://go.dev/doc/effective_go#package-names) and remove the underscore (hyphen in the directory name)?

```suggestion
package logdrop
```